TAGradingServer/account/account-numerictext-gradeable.php

```diff
@@ -21,6 +21,52 @@
     }
 }
 
+if(User::$user_group == 1){
+    $csv_button = "<label>Upload CSV (WARNING! Previously entered data may be overwritten!): Do not include a header row. Format CSV using one column for student id and one column for each field. Columns and field types must match.</br><input type=\"file\" id=\"csvUpload\" accept=\".csv, .txt\" onchange=\"csvUpload()\"></label>";
+    $csv_upload_functions = "
+        function csvUpload(){
+            var f = $('#csvUpload').get(0).files[0];
+            
+            if(f){
+                var reader = new FileReader();
+                reader.readAsText(f);
+                reader.onload = function(evt) {
+                  parseCsv(reader.result);
+                  
+                }
+                reader.onerror = function(evt){
+                    console.error(\"nope\");
+                }
+            } 
+        }
```

Comment:
I'd rather just pass a $_FILES object to be handled by PHP and not JS, especially as we're not really doing anything with the parsed object other than saying "we could open this file!"